api4/user_test.go

```diff
@@ -2261,78 +2163,75 @@ func TestGetUserAudits(t *testing.T) {
 func TestVerifyUserEmail(t *testing.T) {
 	th := Setup().InitBasic()
 	defer th.TearDown()
-	Client := th.Client
 
 	user := model.User{Email: th.GenerateTestEmail(), Nickname: "Darth Vader", Password: "hello1", Username: GenerateTestUsername(), Roles: model.SYSTEM_ADMIN_ROLE_ID + " " + model.SYSTEM_USER_ROLE_ID}
 
-	ruser, _ := Client.CreateUser(&user)
+	ruser, _ := th.Client.CreateUser(&user)
 
 	token, err := th.App.CreateVerifyEmailToken(ruser.Id)
 	if err != nil {
 		t.Fatal("Unable to create email verify token")
 	}
 
-	_, resp := Client.VerifyUserEmail(token.Token)
+	_, resp := th.Client.VerifyUserEmail(token.Token)
 	CheckNoError(t, resp)
 
-	_, resp = Client.VerifyUserEmail(GenerateTestId())
+	_, resp = th.Client.VerifyUserEmail(GenerateTestId())
 	CheckBadRequestStatus(t, resp)
 
-	_, resp = Client.VerifyUserEmail("")
+	_, resp = th.Client.VerifyUserEmail("")
 	CheckBadRequestStatus(t, resp)
 }
 
 func TestSendVerificationEmail(t *testing.T) {
 	th := Setup().InitBasic()
 	defer th.TearDown()
-	Client := th.Client
 
-	pass, resp := Client.SendVerificationEmail(th.BasicUser.Email)
+	pass, resp := th.Client.SendVerificationEmail(th.BasicUser.Email)
 	CheckNoError(t, resp)
 
 	if !pass {
 		t.Fatal("should have passed")
 	}
 
-	_, resp = Client.SendVerificationEmail("")
+	_, resp = th.Client.SendVerificationEmail("")
 	CheckBadRequestStatus(t, resp)
 
 	// Even non-existent emails should return 200 OK
-	_, resp = Client.SendVerificationEmail(th.GenerateTestEmail())
+	_, resp = th.Client.SendVerificationEmail(th.GenerateTestEmail())
 	CheckNoError(t, resp)
 
-	Client.Logout()
-	_, resp = Client.SendVerificationEmail(th.BasicUser.Email)
+	th.Client.Logout()
+	_, resp = th.Client.SendVerificationEmail(th.BasicUser.Email)
 	CheckNoError(t, resp)
 }
 
 func TestSetProfileImage(t *testing.T) {
 	th := Setup().InitBasic()
 	defer th.TearDown()
-	Client := th.Client
 	user := th.BasicUser
 
 	data, err := testutils.ReadTestFile("test.png")
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	ok, resp := Client.SetProfileImage(user.Id, data)
+	ok, resp := th.Client.SetProfileImage(user.Id, data)
 	if !ok {
 		t.Fatal(resp.Error)
 	}
 	CheckNoError(t, resp)
 
-	ok, resp = Client.SetProfileImage(model.NewId(), data)
+	ok, resp = th.Client.SetProfileImage(model.NewId(), data)
 	if ok {
 		t.Fatal("Should return false, set profile image not allowed")
 	}
 	CheckForbiddenStatus(t, resp)
 
 	// status code returns either forbidden or unauthorized
-	// note: forbidden is set as default at Client4.SetProfileImage when request is terminated early by server
-	Client.Logout()
-	_, resp = Client.SetProfileImage(user.Id, data)
+	// note: forbidden is set as default atth.Client..SetProfileImage when request is terminated early by server
```

Comment:
There's a few places here where the find and replace mangled the comments. We might want to fix those

Reply:
Ah, good eye!